geowebcache/web/src/main/resources/log4j2-rollingfile.xml

```diff
@@ -43,4 +43,4 @@
             <AppenderRef ref="Logfile"/>
         </Root>
     </Loggers>
-</Configuration>
\ No newline at end of file
+</Configuration>
```

Comment:
Please keep the newline at the end of file

Reply:
Done 👍